Misc/NEWS.d/next/Library/2020-03-09-01-45-06.bpo-39850.eaJNIE.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`multiprocessing` now supports abstract socket addresses (if abstract sockets
+are supported in the running platform). When creating arbitrary addresses (like when
+default-constructing :class:`multiprocessing.connection.Listener` objects) abstract
+sockets are preferred to avoid the case when the temporary-file-generated address is
```

Comment:
"adddress" here means an absolute path in the filesystem, right?

Reply:
Yep, but that is what is known as the AF_UNIX socket address.